src/libs/Network/enhanceParameters.ts

```diff
@@ -44,7 +35,7 @@ export default function enhanceParameters(command, parameters) {
     finalParameters.api_setCookie = false;
 
     // Include current user's email in every request and the server logs
-    finalParameters.email = lodashGet(parameters, 'email', NetworkStore.getCurrentUserEmail());
+    finalParameters.email = parameters?.email ?? NetworkStore.getCurrentUserEmail();
```

Comment:
I don't think we need the `?` here since parameters is a Record?

Reply:
Agree!